src/components/Avatar/AvatarGroup/AvatarGroup.tsx

```diff
@@ -0,0 +1,82 @@
+import { ComponentElement, ComponentState } from "react"
+import { Avatar, AvatarProps } from ".."
+import { AvatarBase } from "../AvatarBase"
+import styles from "./AvatarGroup.module.css"
+
+type AvatarGroupChildren =
+  | ComponentElement<AvatarProps, ComponentState>
+  | ComponentElement<AvatarProps, ComponentState>[]
```

Comment:
Does this provide value over:
```tsx
type AvatarGroupChildren = 
  | ReactElement<AvatarProps>
  | ReactElement<AvatarProps>[]
```
Just curious.

Reply:
The reason I wasn't using ReactElement is that it doesn't include the name property on `type` which you can see I'm using below where the error is thrown.

Reply:
That's smart 👍